eva/expression/tuple_value_expression.py

```diff
@@ -98,10 +89,6 @@ def signature(self):
             return f"{self.col_object.table_name}.{self.col_object.name}"
         elif isinstance(self.col_object, UdfIOCatalogEntry):
             return f"{self.col_object.udf_name}.{self.col_object.name}"
-        else:
```

Comment:
Same as above. Commenting so that I don't need to revisit.

Reply:
Added `assert`